src/include/duckdb_base.h

```diff
@@ -0,0 +1,641 @@
+//===----------------------------------------------------------------------===//
```

Comment:
lets not do another header file but instead add a define to duckdb.h that can remove the function declarations